### PR TITLE
chore(sentry) - Use new @remix/sentry

### PR DESCRIPTION
**Pull request by SaraVieira**

### CODE REVIEW

**Comment by kof** [comment]
pushed a workaround

File: app/shared/documents/canvas.tsx [hunk 1]
```diff
@@ -6,35 +6,40 @@ import {
...
+          <Env />
+          <Scripts />
+          {process.env.NODE_ENV === "development" && <LiveReload />}
+        </body>
+      </html>
+    </ErrorBoundary>
   );
 };
+
+export const Canvas = withSentryRouteTracing(CanvasRoot);
```
Comment (SaraVieira) [inline app/shared/documents/canvas.tsx:45]: Error happens here, seems to be `@sentry/remix` types not allowing for props but app seems to work the same

Comment (kof) [reply app/shared/documents/canvas.tsx]: withSentryRouteTracing has a wrong props type expectation, it should not be that opinionated I think, we should create an issue there

**Review by kof — APPROVED** [review]
haven't tested

**Comment by SaraVieira** [comment]
@kof updated the sentry version and no more hack needed


Also tested it

**Pull request merged**